tests/modules/nf-core/gatk4/germlinecnvcaller/main.nf

```diff
@@ -68,20 +57,16 @@ workflow test_gatk4_germlinecnvcaller_cohort {
     fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
     fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
     dict = file(params.test_data['homo_sapiens']['genome']['genome_dict'], checkIfExists: true)
+    ploidy_calls = Channel.of([ [ id:'test' ], file(params.test_data['homo_sapiens']['genome']['genome_ploidy_calls'], checkIfExists: true) ])
 
     GATK4_COLLECTREADCOUNTS ( inputs, fasta, fai, dict )
 
-    dgcp_cohort_input = GATK4_COLLECTREADCOUNTS.out.tsv
-            .map({ meta, tsv -> [ [id:'test'], tsv ] })
-            .groupTuple()
-            .combine(bed)
-            .map({ meta, counts, bed -> [ meta, counts, bed, [] ]})
-    GATK4_DETERMINEGERMLINECONTIGPLOIDY_COHORT ( dgcp_cohort_input, priors, [] )
-
     gcnvc_cohort_input = GATK4_COLLECTREADCOUNTS.out.tsv
             .map({ meta, tsv -> return [[id:"test"], tsv ]})
             .groupTuple()
             .combine(bed)
             .map({ meta, counts, bed -> [ meta, counts, bed ]})
-    GATK4_GERMLINECNVCALLER_COHORT ( gcnvc_cohort_input, [], GATK4_DETERMINEGERMLINECONTIGPLOIDY_COHORT.out.calls.collect{ it[1] } )
+    GATK4_GERMLINECNVCALLER_COHORT ( gcnvc_cohort_input, [], ploidy_calls.collect{ it[1] } )
+    GATK4_GERMLINECNVCALLER_COHORT.out.calls
+    GATK4_GERMLINECNVCALLER_COHORT.out.model
```

Comment:
Don't think you need to specify these.

Reply:
If you don't specify these then [this](https://github.com/nf-core/modules/pull/3273#discussion_r1164281781) happens...